x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -449,21 +451,27 @@ async function installPackageByUpload({
   try {
     const { packageInfo } = await generatePackageInfoFromArchiveBuffer(archiveBuffer, contentType);
 
+    // Allow for overriding the version in the manifest for cases where we install
+    // stack-aligned bundled packages. In general, the version pulled from a bundled package
+    // filename and the version in the manifest will match, except for special cases around the
+    // `forceAlignStackVersion` flag in `fleet_packages.json`.
```

Comment:
@kpollich do you not rewrite the manifest with the version, when bundling the package?

Reply:
Good catch - this comment is out of date. We do rewrite the version both in the filename and in the manifest.